lib/ansible/cli/__init__.py

```diff
@@ -613,27 +647,15 @@ def parse(self):
     @staticmethod
     def version(prog):
         ''' return ansible version '''
-        result = "{0} {1}".format(prog, __version__)
-        gitinfo = CLI._gitinfo()
-        if gitinfo:
-            result = result + " {0}".format(gitinfo)
-        result += "\n  config file = %s" % C.CONFIG_FILE
-        if C.DEFAULT_MODULE_PATH is None:
-            cpath = "Default w/o overrides"
-        else:
-            cpath = C.DEFAULT_MODULE_PATH
-        result = result + "\n  configured module search path = %s" % cpath
-        result = result + "\n  ansible python module location = %s" % ':'.join(ansible.__path__)
-        result = result + "\n  executable location = %s" % sys.argv[0]
-        result = result + "\n  python version = %s" % ''.join(sys.version.splitlines())
-        return result
+        # for API compat
+        return _version(prog)
```

Comment:
not sure why you moved all the code to _version ... why not keep here? seems like useless jump